### PR TITLE
feat: crystal 1.3.2

### PR DESCRIPTION
**Pull request by RomanAyupov89**

### CODE REVIEW

File: shard.yml [hunk 1]
```diff
@@ -8,7 +8,6 @@ targets:
   mitm:
     main: src/main.cr
 
-# TODO: Update to "~> 1" once the time comes.
-crystal: ">= 0.36, < 2"
+crystal: 1.3.2
```
Comment (alonchin) [inline shard.yml:11]: why not to keep ">= 1.3.2, < 2". I would like to avoid changing it every time we update crystal

Comment (RomanAyupov89) [reply shard.yml:11]: Just not to forget)

Comment (RomanAyupov89) [reply shard.yml:11]: and for uniformity)

Comment (alonchin) [reply shard.yml:11]: we actually need to update the Crystal version only for engine and tester (high-level app). If there is some issue in other shards we will get this on building the application. 
Moreover, we don't have any specs in mitm which will check that there is no problem with the new Crystal version, so I actually don't see the reason to pin a specific version in our shards.

**Pull request closed without merging**